### PR TITLE
Fix AddLabel to handle no reviews

### PR DESCRIPTION
**Pull request by michelle-yooh**

### CODE REVIEW

**Review by khatwanimohit — APPROVED** [review]

**Review by rwitten — APPROVED** [review]

File: .github/workflows/AddLabel.yml [hunk 1]
```diff
@@ -35,11 +35,27 @@ jobs:
             const repo = "maxtext"
```
Comment (michelle-yooh) [inline .github/workflows/AddLabel.yml:35]: Test comment

File: .github/workflows/AddLabel.yml [hunk 2]
```diff
@@ -17,6 +17,7 @@ name: Add Label
 on:
```
Comment (michelle-yooh) [inline .github/workflows/AddLabel.yml:17]: test comment

File: .github/workflows/AddLabel.yml [hunk 3]
```diff
@@ -17,6 +17,7 @@ name: Add Label
...
   workflow_call:
   pull_request_review:
+  pull_request_review_comment:
```
Comment (michelle-yooh) [inline .github/workflows/AddLabel.yml:20]: test comment

File: .github/workflows/AddLabel.yml (hunk 2 above)
Comment (michelle-yooh) [inline .github/workflows/AddLabel.yml:17]: test comment

**Pull request merged**